### PR TITLE
Fixes bug 1211971 - Rewrote home page template to not use mustache.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@peterbe r?

### CODE REVIEW

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js [hunk 1]
```diff
@@ -91,12 +91,9 @@ $(function() {
...
     },
     buildCrashReports = function(data) {
-        var useTmpl = data.date_range_type === "build" ? crashReportsByBuildDateTmpl : crashReportsByVersionTmpl,
-            crashReportsHTML = Mustache.to_html(useTmpl, data),
-            releaseChannelsContainer = $("#release_channels");
-
-        releaseChannelsContainer.empty().append(crashReportsHTML);
-        releaseChannelsContainer.find('h4').each(function(index, item) {
+        $("#release_channels")
+        .find('h4')
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js]: Not a criticism but why do you do that? I've seen it before. Why not `$('#release_channels h4').each(...`?

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js [hunk 2]
```diff
@@ -91,12 +91,9 @@ $(function() {
...
+        .each(function(index, item) {
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js]: No need for `item`

File: webapp-django/crashstats/crashstats/templates/crashstats/home.html [hunk 3]
```diff
@@ -59,18 +99,16 @@ <h2 id="homepage-heading">{{ product }} {% if version %}{{ version }}{% endif %}
...
 {% endblock %}
 {% block site_js %}
-  {{ super() }}
-  <!--[if IE]><script type="text/javascript" src="{{ static('crashstats/js/flot-0.7/excanvas.pack.js') }}"></script><![endif]-->
-  {% compress js %}
-  <script src="{{ static('crashstats/js/lib/d3.min.js') }}"></script>
-  <script src="{{ static('crashstats/js/lib/metricsgraphics.min.js') }}"></script>
-  {% endcompress %}
+    {{ super() }}
+    <!--[if IE]><script type="text/javascript" src="{{ static('crashstats/js/flot-0.7/excanvas.pack.js') }}"></script><![endif]-->
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/templates/crashstats/home.html]: delete delete delete :)
I know it's off topic but now's the time. We're not using flot any more.

Comment (peterbe) [inline webapp-django/crashstats/crashstats/templates/crashstats/home.html]: Don't forget to `git rm` the actual file too. And check that it's not used anywhere else.

**Comment by peterbe** [comment]
r+ 
Works like a charm. 
Feel free to ignore the comment about deleting that IE tag hack crap. It needs to happen, but you can ignore it in this context.

File: webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js (hunk 1 above)
Comment (AdrianGaudebert) [inline webapp-django/crashstats/crashstats/static/crashstats/js/socorro/dashboard_graph.js]: Haha, that's copy/pasting from the previous mustache template. :)

File: webapp-django/crashstats/crashstats/templates/crashstats/home.html (hunk 3 above)
Comment (AdrianGaudebert) [inline webapp-django/crashstats/crashstats/templates/crashstats/home.html]: Bad news: it's still being used in 3 places, explosive crashes, status and topcrasher.

**Pull request merged**